App.js

```diff
@@ -24,7 +24,33 @@ Folosește React pentru a afișa următorul cod în elementul cu ID #app
 */
 
 export default function App() {
-    return (
-        <div></div>
-    )
-}
\ No newline at end of file
+  const shoppingListItems = [
+    "Pâine",
+    "Lapte",
+    "Ciocolată",
+    "Mere",
+    "Miere",
+    "Mandarine",
+  ];
+  return (
+    <div className="vh-100 d-flex flex-column">
+      <header className="navbar bg-primary bg-gradient">
+        <div className="container-fluid">
+          <h1 className="navbar-brand mb-0 text-white">Tema React</h1>
+        </div>
+      </header>
+      <main className="container my-5 flex-grow-1 d-flex flex-column align-items-center justify-content-center">
+        <ul className="list-group">
+          <li className="list-group-item">
+            <span className="fw-bold">Lista de cumpărături</span>
+          </li>
```

Comment:
Nu e asta varianta finală, încerc să mă gandesc la o metodă prin care să includ si primul element din listă în map. Lucrez în continuare și revin.

Reply:
Nu e nevoie să fie absolut toate elementele în listă. E OK așa.

Reply:
Am reușit până la urma cu condiție, sper că e okay și așa.  Voiam să vad dacă-mi iese. 😄